lib/config/config-loader.js

```diff
@@ -166,39 +237,33 @@ async function loadConfigFile(filePath) {
      *
      * When Node.js supports native TypeScript imports, we can remove this check.
      */
-    if (isTS && !isDeno && !isBun) {
-
-        // eslint-disable-next-line no-use-before-define -- `ConfigLoader.loadJiti` can be overwritten for testing
-        const { createJiti } = await ConfigLoader.loadJiti().catch(() => {
-            throw new Error("The 'jiti' library is required for loading TypeScript configuration files. Make sure to install it.");
-        });
-
-        // `createJiti` was added in jiti v2.
-        if (typeof createJiti !== "function") {
-            throw new Error("You are using an outdated version of the 'jiti' library. Please update to the latest version of 'jiti' to ensure compatibility and access to the latest features.");
-        }
 
-        /*
-         * Disabling `moduleCache` allows us to reload a
-         * config file when the last modified timestamp changes.
-         */
+    if (isTS) {
+        if (hasUnstableNativeNodeJsTSConfigFlag) {
+            if (isNativeTypeScriptSupportEnabled()) {
+                return await dynamicImportConfig(filePath, fileURL, mtime);
+            }
 
-        const jiti = createJiti(__filename, { moduleCache: false, interopDefault: false });
-        const config = await jiti.import(fileURL.href);
+            // eslint-disable-next-line n/no-unsupported-features/node-builtins -- it's still an experimental feature.
+            if (typeof process.features.typescript === 'undefined') {
```

Comment:
We could use the `in` operator to avoid the rule violation:

```suggestion
            if (!('typescript' in process.features)) {
```

Reply:
Done!